server/src/main/java/org/elasticsearch/index/mapper/RootObjectMapper.java

```diff
@@ -106,12 +106,33 @@ public RootObjectMapper.Builder addRuntimeFields(Map<String, RuntimeField> runti
 
         @Override
         public RootObjectMapper build(MapperBuilderContext context) {
+            Map<String, Mapper> mappers = buildMappers(context);
+            Map<String, Mapper> aliasMappers = new HashMap<>();
```

Comment:
maybe move this logic to a separate method? Then this can also be unit tested in isolation.

Reply:
Let's add a helper method that returns `aliasMappers`?